doozer/tests/lockfile_prototype/test_shell_parser.py

```
@@ -113,6 +113,23 @@ def test_arch_conditional_x86_64(self):
         self.assertEqual(common, ["make"])
         self.assertEqual(arch, {"x86_64": ["kernel-rt-devel", "kernel-rt-modules"]})
 
+    def test_reinstall_packages_parsed(self):
+        run_values = ["dnf -y reinstall tzdata && dnf clean all"]
+        common, arch = extract_packages_from_run_commands(run_values)
```

Ruff (0.15.21)

[warning] Unpacked variable `arch` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)